ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -354,6 +441,14 @@ open class SegmentedControl: UIControl {
         invalidateIntrinsicContentSize()
     }
 
+    @objc open func getViewForButton(at index: Int) -> UIView? {
```

Comment:
something about this name is un-natural for me in swift. first it isn't a button we are returning so "forbutton" doesn't make sense.